src/Goods/Goods.jsx

```diff
@@ -0,0 +1,7 @@
+export const Goods = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li data-cy="Good">{good}</li>
```

Comment:
Each list item should have a unique `key` prop to help React identify which items have changed, been added, or removed. Consider using a unique identifier from the `good` or its index in the array as the `key`.